crates/iota-graphql-rpc-client/src/response.rs

```diff
@@ -40,6 +40,7 @@ impl GraphqlResponse {
         })
     }
 
+    #[allow(clippy::result_large_err)]
     pub fn graphql_version(&self) -> Result<String, ClientError> {
```

Comment:
I am curious why we did not use Box in `iota-graphql-rpc-client` as opposed to the rest of the code to fix the same problem `result_large_err` ? 
This result_large_err - https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err - is under perf label, looks like it effects compile time perf, probably not runtime perf. Using Box will probably effect runtime perf, but it might be negligible most times. Of course, all of these matter only if these code sections are perf critical or hot path etc..

Reply:
It comes down to personal preferences of the different codeowners. Here the infra team preferred to ignore it as it's not significantly above the threshold, 168 bytes with a lint at 128 bytes. For comparison, the `UnexpectedEventFormat` is at 320 bytes. `ObjectFoundWhenPackageExpected` could have been ignored as it's "only" at 152 bytes though.